src/web/app/pages-instructor/instructor-course-details-page/instructor-course-details-page.component.ts

```diff
@@ -81,19 +84,94 @@ export class InstructorCourseDetailsPageComponent implements OnInit {
   }
 
   /**
-   * Loads the course's details based on the given course ID and email.
+   * Loads the course's details based on the given course ID.
    */
   loadCourseDetails(courseid: string): void {
-    const paramMap: { [key: string]: string } = { courseid };
-    this.httpRequestService.get('/courses/details', paramMap).subscribe((resp: CourseInfo) => {
```

Comment:
And you can also remove the old endpoint?

Reply:
Alright